PiwikTracker.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |spec|
   spec.name         = "PiwikTracker"
-  spec.version      = "3.3.0"
-  spec.summary      = "A Piwik tracker written in Objective-C for iOS and OSX apps."
+  spec.version      = "4.0.0-alpha1"
+  spec.summary      = "A Piwik tracker written in Swift for iOS and OSX apps."
```

Comment:
I would say we start with iOS and add macOS and tvOS later. I don't think it will be a huge issue though.